scripts/unused-key-cleanup/cleanup-form-model.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable */
+
+/*
+Map emails array to proper structure
+*/
+
+// Check total forms count with responseMode email
+db.getCollection('forms').find({ responseMode: 'email' }).count()
+
+// Check total forms count with responseMode email and emails as an array 
+// ~ Should be the same number as the total number of email mode forms
+db.getCollection('forms').find({ responseMode: 'email', "emails" : { $type : "array" } }).count()
+
+// !!!! MAIN UPDATE SCRIPT !!!!
+
+// Cases
+// ['test@hotmail.com'] => ['test@hotmail.com'] ~ unchanged
+// ['test@hotmail.com', 'test@gmail.com'] => ['test@hotmail.com', 'test@gmail.com'] ~ unchanged
+// ['test@hotmail.com, test@gmail.com'] => ['test@hotmail.com', 'test@gmail.com']
+// ['test@hotmail.com, test@gmail.com', 'test@yahoo.com'] => ['test@hotmail.com', 'test@gmail.com', 'test@yahoo.com']
+
```

Comment:
just to check - are there any semicolons (;) present in the database?

Reply:
yes, there are. updated the script to account for this. Thanks!